models/pwa.js

```diff
@@ -22,7 +22,7 @@ class Pwa {
   constructor(manifestUrl, manifestModel) {
     this.manifestUrl = manifestUrl;
     this.manifest = manifestModel;
-    this.lighthouseScore = 0;
+    this.lighthouseScore = undefined;
```

Comment:
We can probably remove this line, since this.lighthouseScore will be undefined by default.